src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -74,7 +76,13 @@ public function getAction($id, Request $request)
             $view = $this->responseGetById(
                 $id,
                 function ($id) use ($locale, $mediaManager) {
-                    return $mediaManager->getById($id, $locale);
+                    $media = $mediaManager->getById($id, $locale);
+
+                    if ($media->getEntity()->getCollection()->getType()->getKey() === 'collection.system') {
```

Comment:
Isn't the `collection.system` string already stored somewhere? I am already duplicating it...